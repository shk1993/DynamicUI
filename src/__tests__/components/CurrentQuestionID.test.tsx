import * as React from 'react';
import { render } from 'react-testing-library';
import CurrentQuestion from '../../components/CurrentQuestionID';

describe('The current question component', () => {
  it('should render and show the current question ID', () => {
    const { container } = render(<CurrentQuestion currentQuestion="DE1" />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render a different id when passed a different prop', () => {
    const { getByText } = render(<CurrentQuestion currentQuestion="DE2" />);
    expect(getByText('DE2', { exact: false })).toMatchSnapshot();
  });
});
