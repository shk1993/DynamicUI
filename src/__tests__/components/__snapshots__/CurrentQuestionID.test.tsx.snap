// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The current question component should render a different id when passed a different prop 1`] = `
<span
  style="color: white;"
>
  [
  DE2
  ]
</span>
`;

exports[`The current question component should render and show the current question ID 1`] = `
<span
  style="color: white;"
>
  [
  DE1
  ]
</span>
`;
